packages/KCard/KCard.vue

```diff
@@ -66,7 +71,6 @@ export default {
 
 .kong-card {
   padding: var(--KCardPaddingY, 1rem) var(--KCardPaddingX, 1rem);
-  margin-bottom: var(--KCardMarginY, 1rem) var(--KCardMarginX, 1rem);
```

Comment:
![image](https://user-images.githubusercontent.com/52717686/73097141-4d8a8c80-3e9b-11ea-99e3-6a3eb7a20a31.png)


@xitongzou fixed `margin-bottom` for above, but it seems should be `margin-bottom: var(--KCardMarginY, 1rem) ;`
(https://github.com/Kong/kongponents/pull/185/files#diff-870dbd37faad1d06526ecdc0d78ca840R69)

Reply:
I removed this because cards shouldn't worry about margin. If a user wants to add margin then they can do it like:

`<KCard class="mb-2">`

This helps with composition.

Reply:
ok, I will create a PR to KM, after this got merge.

Reply:
yea we'll need to review some things such as immunity alert cards as well